server/src-bench-cache/Main.hs

```diff
@@ -0,0 +1,281 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-} 
+module Main where
+
+import Prelude
+import Criterion.Main
+import Data.Word
+import Data.Bits
+import Data.IORef
+import Data.List
+import Data.List.Split (chunksOf)
+import Data.Ord
+import Data.Traversable
+import qualified Data.Vector as V
+import Control.Concurrent.Async (forConcurrently)
+import Control.Concurrent (getNumCapabilities)
+import Control.DeepSeq
+import Control.Monad (foldM)
+import GHC.Clock
+import System.Random.MWC as Rand
+import System.Random.MWC.Probability as Rand
+import qualified Hasura.Cache.Bounded   as B
+import qualified Hasura.Cache.Unbounded as U
+-- higher level interface to above, combined:
+import qualified Hasura.Cache as Cache
```

Comment:
Minor point, but: could you run `stylish-haskell` on this file?

Reply:
Just made those changes, thanks!